tests/test_score.py

```diff
@@ -388,3 +388,18 @@ def test_score_pairs_age(self):
         self.assertEqual(len(results), 1)
 
         self.assertAlmostEqual(float(results[0][2]), 1.0, 1)
+
+    def test_alpha_zero(self):
+        """the root term should contain all diseases therefore the IC should be zero"""
+        root_term_ic = self.hpo_network.nodes['HP:0000118']['ic']
+        self.assertEqual(0.0,root_term_ic)
+
+    def test_leaves_diff_branches_score_zero(self):
+        """two leaves in different branches
+        two leaves therfore beta is zero
+        different branches therefore alpha is zero
+        define I = (0.0 / (0.0 + 0.0)) as zero and not nan"""
+        generalized_hypotonia_term = 'HP:0001290'
+        moderate_receptive_langage_delay_term = 'HP:0011351'
+        score_two_leaves_diff_branches = self.scorer.score_hpo_pair_hrss(generalized_hypotonia_term,moderate_receptive_langage_delay_term)
```

Comment:
To keep with the style of other tests, I would name them term_a, term_b and add a comment with what they are instead of naming the variable.